examples/iot_lock.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: IoT Lock Controller
+parent: Example Applications
+has_children: false
+nav_order: 2
+---
+
+# IoT - Tupelo Based Lock Control via Zwave and Raspberry Pi
+{: .fs-9 }
+
+This was built by Nathan McDowell the winner of a Tupelo coding challenge.
+This proof of concept IoT project uses ChainTrees, Zwave and Raspberry Pi to
+remotely control a house lock.  
+
+The react-native client can be granted temporary access with control and tracking
+all signed by the Tupelo testnet.
+
+<img style="width:515px;height:403px; padding: 15px;" src="tupelo-zwave.gif">
+
+There are wide applications for Tupelo for IoT because it is efficient,
+fast and even includes the possibility of devices being able to check transaction
+validity when they are offline.  A small example of offline functionality
+[can be found here](https://github.com/quorumcontrol/offliner) but hop into our
+developer chat or ask for more information if you have any questions.
```

Comment:
link to chat.

Reply:
I should have just done it then pushed it but... this change is in the next iteration branch improving-examples .